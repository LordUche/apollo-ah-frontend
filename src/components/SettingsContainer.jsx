/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';

import Settings from '../views/Settings';
import ProfileHeaderContainer from './ProfileHeaderContainer';

/**
 * Container component for the Settings view
 * @export
 * @class SettingsContainer
 * @extends {Component}
 */
export default class SettingsContainer extends Component {
  render() {
    return (
      <>
        <ProfileHeaderContainer activePage="SETTINGS" />
        <Settings />
      </>
    );
  }
}
