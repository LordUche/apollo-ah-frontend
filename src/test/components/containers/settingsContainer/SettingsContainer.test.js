import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SettingsContainer from '../../../../components/containers/SettingsContainer';

configure({ adapter: new Adapter() });

describe('<SettingsContainer>', () => {
  it('should render', () => {
    const wrapper = shallow(<SettingsContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
