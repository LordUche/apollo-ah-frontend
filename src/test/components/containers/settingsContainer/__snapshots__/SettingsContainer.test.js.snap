// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsContainer> should render 1`] = `
<Fragment>
  <Connect(ProfileHeaderContainer)
    activePage="SETTINGS"
  />
  <Settings />
</Fragment>
`;
